src/task/ReceiveVideoStreamIndexTask.ts

```diff
@@ -206,6 +213,43 @@ export default class ReceiveVideoStreamIndexTask
     }
   }
 
+  private handleSupportedVideoReceiveCodecIntersection(index: SdkIndexFrame): void {
+    if (this.context.videoSendCodecPreferences === undefined) {
+      return;
+    }
+
+    const newMeetingSupportedVideoSendCodecPreferences: VideoCodecCapability[] = [];
+    let willNeedUpdate = false;
+
+    // Interesect `this.context.videoSendCodecPreferences` with `index.supportedReceiveCodecIntersection`
+    for (const capability of this.context.videoSendCodecPreferences) {
+      for (const signaledCapability of index.supportedReceiveCodecIntersection) {
+        if (capability.equals(VideoCodecCapability.fromSignaled(signaledCapability))) {
+          newMeetingSupportedVideoSendCodecPreferences.push(capability);
+          break;
+        }
+      }
+      // We don't need to renegotiate if we aren't sending a codec that is no longer supported in the call.
+      if (
+        this.context.currentVideoSendCodec &&
+        capability.equals(this.context.currentVideoSendCodec)
+      ) {
+        willNeedUpdate = true;
```

Comment:
Not sure if this is correct but I don't think the code here matches the comment.

Reply:
Will remove the double negative. Comment was referring to the non-existent `else` branch.